docs/samples/README.md

```diff
@@ -9,6 +9,7 @@ Knative code samples that are actively tested and maintained by Knative working
 
 - [Eventing and Eventing Sources code samples](eventing.md)
 - [Serving code samples](serving.md)
+- [E2E Sample Application - Knative Bookstore](https://github.com/knative/docs/tree/main/code-samples/eventing/bookstore-sample-app)
```

Comment:
Nit: maybe we can indicate that this uses serving and eventing and functions? The other items in this list mention serving/eventing so it can be a little confusing to figure out which is used from just the text here